cerberus/tests/test_validation.py

```diff
@@ -1,14 +1,18 @@
 # -*- coding: utf-8 -*-
 
-import re
-import sys
+
+from bson import decimal128
```

Comment:
to repeat myself: would it suffice to test that in Eve? i'd rather avoid extra dependencies, even for testing.